src/db/models/art.js

```diff
@@ -7,6 +7,7 @@ module.exports = (sequelize, DataTypes) => {
     description: DataTypes.TEXT,
     categoryId: DataTypes.INTEGER,
     featuredImg: DataTypes.STRING,
+    price: DataTypes.DECIMAL(10, 2),
```

Comment:
can you please explain this line

Reply:
It means save the data in two decimal points. This is mostly used fields that hold currencies

Reply:
okay